apps/src/code-studio/pd/foorm/FoormEditorManager.jsx

```diff
@@ -18,6 +18,7 @@ const styles = {
 class FoormEditorManager extends React.Component {
   static propTypes = {
     updateFormQuestions: PropTypes.func,
```

Comment:
Should the `updateFormQuestions` prop be removed? Or is it used elsewhere?

Reply:
Oh it looks like it's still being passed as a prop in apps/src/sites/studio/pages/foorm/editor/index.js -- maybe just help me understand why it's sticking around and most other places language like FormQuestions is being swapped out for FormData?

Reply:
good question. The form data is an object in the form {published: true/false, questions: {entire json blob of survey}}, while form questions is the json blob of the survey. We get the form data once when loading a survey, then send around the form questions as the survey author edits them. I can add a comment somewhere to make this clearer.